docs/admin/disruptions.md

```diff
@@ -0,0 +1,87 @@
+---
+assignees:
+- mml
+
+---
+This guide is for anyone wishing to specify safety constraints on pods or anyone
+wishing to write software (typically automation software) that respects those
+constraints.
+
+* TOC
+{:toc}
+
+## Rationale
+
+Various cluster management operations may "voluntarily" evict pods. By
```

Comment:
Avoid using "we" as a general principle. Suggested rephrase:

"In a "voluntary" eviction , a pod is removed from a node in a controlled manner, such as draining the node for maintenance (`kubectl drain`) or autoscaling a cluster downward. In the future, the [rescheduler] may also perform voluntary evictions. By contrast, a non-voluntary eviction removes a pod because a node has become unreachable or reports NotReady."